NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/TransferGroup.java

```diff
@@ -52,12 +52,6 @@ public boolean execute(CommandSender sender, String[] args) {
 			return true;
 		}
 		
-		if (!gm.hasAccess(g, uuid, PermissionType.getPermission("TRANSFER")) && 
```

Comment:
I removed the whole permission, because it didnt make any sense. By definition of the functionality only the primary owner of a group could transfer it (which was always enforced through an additional check), so having this permission was completly pointless